wissen: add akteure english (#53)
ait-energy/en.ergie.at: content/wissen/akteure/index.en.md

## Balancing Group Representative ("Bilanzgruppenverantwortlicher")

Each balancing group has a balancing group representative who represents it towards other market participants. The balancing group representative coordinates the forecasts for generation and consumption within the balancing group and prepares schedules based on them. They also receive the settlement of the balancing energy from the balancing group coordinator and settle this with the members of the balancing group.[^1]

## Network Operator ("Netzbetreiber")

Network operators are responsible for the reliable transport of electrical energy and the operation of the electricity grids. They ensure that electricity flows from generation plants to consumers by managing grid operations, providing the necessary infrastructure, and handling grid charges. Since the electricity grid constitutes a natural monopoly, network operators are subject to strict regulation by the [regulatory authority](#regulierungsbehörde) E-Control. A distinction is made between transmission system operators for large-scale electricity transport and distribution system operators for regional electricity distribution.[^1]

> #### Transmission System Operator ("Übertragungsnetzbetreiber")

Reviewer: ```suggestion
### Transmission System Operator ("Übertragungsnetzbetreiber")
```

Transmission system operators (TSOs) operate the [extra-high and high voltage grid](/wissen/stromnetz/) and ensure the transport of large volumes of electricity over long distances to regional distributors. They also handle cross-border electricity transit. In Austria, this task is performed by [APG](/wissen/stromnetz/), which operates around 7,000 km of transmission lines.[^1] [^2]

> #### Distribution System Operator ("Verteilnetzbetreiber")

Reviewer: ```suggestion
### Distribution System Operator ("Verteilnetzbetreiber")
```

> Distribution system operators (DSOs) take over electricity from the transmission system and distribute it via [medium and low voltage lines](/wissen/stromnetz/) to households, businesses, and other end consumers. In Austria, there are currently 114 electricity distribution system operators operating a total of around 260,000 km of lines.[^3]

Reviewer: ```suggestion
Distribution system operators (DSOs) take over electricity from the transmission system and distribute it via [medium and low voltage lines](/wissen/stromnetz/) to households, businesses, and other end consumers. In Austria, there are currently 114 electricity distribution system operators operating a total of around 260000 km of lines.[^3]
```

Reviewer: Just doing that because some non-native people tend to misread the `,`.
